### DISCUSSION TITLE
Add name if lookuped into other_1 field. But......

### DISCUSSION THREAD

**Discussion by DE8MSH** [mbridak/not1mm#61] [Q&A]
I added this line (green):

![image](https://github.com/mbridak/not1mm/assets/2323972/f45902b5-d786-4419-88b6-6a96b970b279)

But it creates a fine segmentation fault. Uhm... Why?

**Comment by mbridak** [comment]
That's a good question. I don't see a reason for it to segfault.
But I would caution you against making field setting code in __main__.py

Only because the fields other_1 and other_2 are often modified and read by the contest plugins. They serve different purposes based on the contest chosen. You could nest it in code that makes sure it's running the general logging plugin. something like:

```
if self.contest:
    if self.contest.name ==  "General Logging":
        self.other_1.triggerCrash(True)
```

But back to the original question.... I really don't know.

**Comment by DE8MSH** [comment]
hmmmm. five lines deeper i use nick, too. no probs there. only if i settext to other_1 -- and i guess tabbing to name field (other_1) and focussing it. to me it makes no sense to xtract name/nickname and dont set it to other_1 (general logging). because of typing it manually from a to b ;)
i dunno why its a difference if i type it manually or script does it.

**Comment by mbridak** [comment]
Yeah it's not the nick variable.

I inserted:

```python
print(f"{type(self.other_1)}")
self.other_1.setText("test")
```

which resulted in:

```
<class 'PyQt5.QtWidgets.QLineEdit'>
Segmentation fault (core dumped)
```

So at the time self.other_1 is still a valid QLineEdit object, and I'm just passing in a plain vanilla string object. 
As far as I can tell it's crashing in the PyQt library.
